tools/FakeItEasy.Build/Program.cs

```diff
@@ -80,8 +81,8 @@ public static void Main(string[] args)
                 Target(
                     testSuite.Key,
                     DependsOn("build", "testsDirectory"),
-                    forEach: testSuite.Value,
-                    action: testDirectory => RunTests(testDirectory));
+                    forEach: testSuite.Value.SelectMany(project => GetFrameworks(project).Select(framework => new { Project = project, Framework = framework })),
+                    action: tests => RunTests(tests.Project, tests.Framework));

```

Comment:
Oh, and a bit of nitpicking: this should be `test`, not `tests`, since it represents a single test project 😉

Reply:
I'm not sure I agree. The name of the methods is `RunTests()`, since it does indeed run a number of tests. All those tests belong to some project, and are run using some framework.

Reply:
Fair point. As I said, it was nitpicking ;)